OpenRA.Mods.Cnc/Traits/PortableChrono.cs

```diff
@@ -55,6 +55,18 @@ class PortableChronoInfo : ITraitInfo, Requires<IMoveInfo>
 		[VoiceReference]
 		public readonly string Voice = "Action";
 
+		[Desc("Range circle color")]
```

Comment:
I generally prefer descriptions to end with a full stop, especially when the traits have other fields that already have one (which should be the case most of the time, so it would be best if you just added it to all descriptions of the new fields).

Reply:
Only sentences end with a full stop.